src/app/components/common/AppBar/components/NetworkSwitchToggle/NetworkOption.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Grid from '@mui/material/Grid';
+import { Typography } from '@mui/material';
+import { changeCurrentNetwork } from '~lib/utils/envHelper';
+import { useStyles } from '~app/components/common/AppBar/components/NetworkSwitchToggle/NetworkToggle.styles';
+
+type NetworkOptionProps = {
+    networkId: number;
+    optionLabel: string;
+};
+
+const NetworkOption = ({ networkId, optionLabel }: NetworkOptionProps) => {
+    const changeNetworkHandler = () => {
+        changeCurrentNetwork(networkId);
```

Comment:
Is change current network a hook or simple function?

Reply:
just a function